skore/src/skore/sklearn/cross_validation/plot.py

```diff
@@ -56,7 +60,7 @@ def plot_cross_validation(cv_results: dict) -> plotly.graph_objects.Figure:
         bar_color = plotly.colors.qualitative.Plotly[
             col_i % len(plotly.colors.qualitative.Plotly)
         ]
-        bar_x = [min(df.index) - 0.5, max(df.index) + 0.5]
+        bar_x = numpy.linspace(min(df.index) - 0.5, max(df.index) + 0.5, num=10)
```

Comment:
Do you think we can do the same using only stdlib?

Reply:
sure -- numpy is in our transitive dependencies (because of sklearn) but I can rework it

Reply:
done

Reply:
Nice, thanks!